mindsdb/integrations/postgres_handler/postgres_handler.py

```diff
@@ -0,0 +1,101 @@
+from mindsdb.integrations.libs.base_handler import DatabaseHandler
+import psycopg
+from mindsdb.utilities.log import log
+from contextlib import closing
+from mindsdb_sql import parse_sql
+
+class PostgresHandler(DatabaseHandler):
+    """
+    This handler handles connection and execution of the PostgreSQL statements. 
+    """
+
+    def __init__(self, name, **kwargs):
+        super().__init__(name)
+        self.parser = parse_sql
+        self.dialect = 'postgresql'
+        self.database = kwargs.get('database')
+        self.host = kwargs.get('host')
+        self.port = kwargs.get('port')
+        self.user = kwargs.get('user')
+        self.password = kwargs.get('password')
+
+    def __connect(self):
+        """
+        Handles the connection to a PostgreSQL database insance.
+        """
+        #TODO: Check psycopg_pool
+        connection = psycopg.connect(f'host={self.host} port={self.port} dbname={self.database} user={self.user} password={self.password}', connect_timeout=10)
```

Comment:
May be we should save all argument to dict and use it as dict? So we will be able use any connection parameter (as i remember someone already asked about ssl mode https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PARAMKEYWORDS). 
```
def __init__(self, name, **kwargs):
    self.connection_args = kwargs
    self.connection_args['dbname'] = self.connection_args['database']
    del self.connection_args['database']    $ or map it in __connect ?
    
def __connect(self):
    connection = psycopg.connect(**self.connection_args, connect_timeout=10)
```
will be need map 'database' to 'dbname